### DISCUSSION TITLE
Can we remove the /v/ in pages /v/some-page?

### DISCUSSION THREAD

**Discussion by romosa** [GitbookIO/community#377] [0. General bugs and questions]
Hello,

We are setting up some collection and noticed some behavior where it /v/ before the collection.

I have this configuration,

Collection
 - page1
 - page2
 - page3
 
Visiting page 1 will be like `domain.com/collection/page1`  but for the other page there will /v/ appended. 
```
domain.com/collection/v/page2
domain.com/collection/v/page3

```

Is there a way to change this behavior to become like this?
```
domain.com/collection/page1 
domain.com/collection/page2
domain.com/collection/page3
```

**Comment by aikoii** [comment]
👋 @romosa! I'm afraid, you won't be able to remove the "v" in the path, as the URL will include a /v/ portion before the space path. This is how we address variants spaces under a [Collection](https://docs.gitbook.com/publishing/share/collection-publishing).
